lib/init/npm-utils.js

```diff
@@ -119,12 +119,12 @@ function check(packages, opt) {
         throw error;
     }
 
-    if (opt.devDependencies && typeof fileJson.devDependencies === "object") {
-        deps = deps.concat(Object.keys(fileJson.devDependencies));
-    }
-    if (opt.dependencies && typeof fileJson.dependencies === "object") {
-        deps = deps.concat(Object.keys(fileJson.dependencies));
-    }
+    ["dependencies", "devDependencies"].forEach(key => {
+        if (opt[key] && typeof fileJson[key] === "object") {
+            deps = deps.concat(Object.keys(fileJson[key]));
+        }
+    });
+
     return packages.reduce((status, pkg) => {
         status[pkg] = deps.indexOf(pkg) !== -1;
```

Comment:
By the way, can you refactor it as below? Thanks.

```suggestion
         status[pkg] = deps.includes(pkg);
```